spec/helpers/applications_helper_spec.rb

```diff
@@ -85,6 +85,12 @@
     end
   end
 
+  describe "decode" do
+    it "should decode the html entities such as &amp; to &" do
```

Comment:
I tried using <script></script> both encoded and decoded, and both cases browser treated it like a text. Can you think of any other ways  of testing it?